test/DynamoCoreWpfTests/RecordedTests.cs

```diff
@@ -1155,6 +1155,62 @@ public void TestReconnectionUndo()
             Assert.AreEqual(2, workspace.Connectors.Count()); 
         }
 
+        /// <summary>
+        /// The following recorded test manually adds a connection between 2 nodes,
+        /// followed by ctrl + clicking to copy this connection to 2 additional nodes.
+        /// The final assertion will only be true if all connections were successfully established.
+        /// </summary>
+        [Test, RequiresSTA]
```

Comment:
just curious, what happens if you remove the requiresSTA?

Reply:
doesn't seem to matter but is used in all the previous implementations, I can remove from the tests I added